server/api/tests/test_schemas.py

```diff
@@ -0,0 +1,161 @@
+import json
+import logging
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+from graphql_schemas.schema import schema
+
+logging.disable(logging.ERROR)
+
+
+class context(object):
+    def __init__(self, user_id):
+        self.user = User(id=user_id)
+
+
+class AttendanceTestCase(TestCase):
+    fixtures = ['initial.json']
```

Comment:
I don't understand what this does. If you have the time, please explain.

Reply:
@tomipaul I would advice that will avoid using seeders in unit testing reason been that using fixtures are hard to maintain as a project’s data changes over time. Modifying JSON-formatted files to match our last migration is hard, especially as it can be difficult to identify during the JSON load process where our JSON  fils is either broken or inaccurate representation of the database.

we can use the following in place:-
Factory boy:  A package that generates model test data.
Model mommy:  Another package that generates model test data.

Reply:
As @tosmak16 has indicated, it merely seeds data into the test database prior to running test cases.
I very much agree with @tosmak16 on this though, it can become more and more difficult to maintain as model changes. I think we should discuss this at the team level so we can all be on the same page